### DISCUSSION TITLE
Rainbow kit

### DISCUSSION THREAD

**Discussion by SargsyanGegham** [rainbow-me/rainbowkit#2081] [Feedback]
Wallet connect not working in telegram bot mini app do you have any suggestions or solution for this problem? (In telegram bot mini app my project opens inside webview component and windows object is not same with user phone context ) if you have any solution please let me know

**Comment by 0xmkh** [comment]
@SargsyanGegham Sounds like a wallet deep link issue. Can you share a reproducible example ?

**Comment by SargsyanGegham** [comment]
Thanks for your reply. yes it doesn't work in Telegram Bot Min app, the
problem is in your connection logic, it only works in the top context where
window.ethereum is not undefined.

вс, 28 июл. 2024 г. в 02:29, MK ***@***.***>:

**Reply by 0xmkh** [reply]
Do you have a minimal repro example ?

**Comment by SargsyanGegham** [comment]
Unfortunately I can’t share my code, but you can try open your connection
logic inside of telegram bot, I think for resolving this problem you need
use deep links, anyway if you want replicate this issue you should create
test

**Comment by SargsyanGegham** [comment]
FYI connectKit work for ios, I customize and fix connection for both
android and iOS devices
